### PR TITLE
Whatcompmarg

### LINKED ISSUE #88: What Component Margining
image

The title, text, and button is a little too closely spaced, add a little bit of margining

### PR DESCRIPTION
**Pull request by zalam007**
wc

### CODE REVIEW

File: src/components/home/What.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ const What = () => {
   return (
     <div className="flex flex-col items-center justify-center my-[5%]">
       <Title text="What We Do" />
-      <p className="text-center md:text-2xl text-xl my-3 w-3/4">
+      <p className="text-center text-2xl my-5 w-[90%]">
```
Comment (seanquiambao) [inline src/components/home/What.jsx]: revert back the original lines of code and only change margining. we need these lines of code for mobile responsiveness. you might have forgotten to pull from development branch.

File: src/app/board/page.js
Comment (seanquiambao) [inline src/app/board/page.js:1]: unstage the board/page.js changes from this branch, this is a separate issue.

**Review by seanquiambao — CHANGES_REQUESTED** [review]

**Comment by zalam007** [comment]
made changes, should be repushed

File: src/app/board/page.js
Comment (seanquiambao) [inline src/app/board/page.js:1]: the changes in the /app/board/page.js needs to be removed from this branch

**Review by seanquiambao — CHANGES_REQUESTED** [review]
address some merge conflicts as well

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**